tasks/install-elasticsearch.yml

```diff
@@ -3,7 +3,7 @@
     - install
     - upgrade-charm
     - config-changed
-  apt_key: url={{ apt_key_url }} state=present id={{gpg_key_id}}
+  apt_key: url={{ apt_key_url }} state=present id={{gpg_key_id}} validate_certs=no
```

Comment:
@marcoceppi you can simply check it yourself in the browser https://artifacts.elastic.co/GPG-KEY-elasticsearch

The apt-key-url changed from pre 5. releases, you can see its not even wrapped in anything here http://packages.elasticsearch.org/GPG-KEY-elasticsearch